portal-ui/public/precache-manifest.d1c8f4292065174e51a3d57ce7578ff7.js

```diff
@@ -0,0 +1,30 @@
+self.__precacheManifest = (self.__precacheManifest || []).concat([
```

Comment:
I think you checked in the compiled UI. Let's not do that, as the UI is built inside the container